tools/Localizations/Sources/Localizations/main.swift

```diff
@@ -75,13 +75,53 @@ struct L10NTools: ParsableCommand {
         default: return true;
         }
     }
-    
+
+    private func getLocalesFromProjectFolder () -> (Array<String>) {
+        var myLocalesList:[String] = []
+
+        let blockzillaFolder = FileManager.default.enumerator(atPath: URL(fileURLWithPath: projectPath).deletingLastPathComponent().appendingPathComponent("Blockzilla").path)
+
+        let filePaths = blockzillaFolder?.allObjects as! [String]
+        let textFilePaths = filePaths.filter{$0.contains(".lproj")}
+        for txt in textFilePaths {
+            if let index = txt.firstIndex(of: ".") {
+                let firstPart = txt.prefix(upTo: index)
+                myLocalesList.append(String(firstPart))
+            }
+        }
+        var uniqueLocales = Array(Set(myLocalesList))
+        
+        let toRemove = "Settings"
+        
+        uniqueLocales = uniqueLocales.filter { $0 != toRemove }
+
+        for item in uniqueLocales {
+            print(item)
+            for (key, _) in locale_mapping {
+                if item == key {
+                    let position = uniqueLocales.firstIndex(of: item)!
+                    uniqueLocales[position] = locale_mapping[key]!
+                }
+            }
+        }
+
+        return uniqueLocales.sorted(by:<)
+    }
+
+    private let locale_mapping = [
```

Comment:
This is the way I found so that the locales name is correct and the import works